### PR TITLE
Romanian (ro) translate

### PR DESCRIPTION
**Pull request by gilav23**
First of all thanks for the app, it is an option for call recording even without root.
I am new to this but want to help to develop the app. I have inserted the translation into the file ‎app/src/main/res/values-ro/strings.xml. I also updated the app/src/main/res/xml/locales_config.xml and app/build.gradle.kts files.
If I got something wrong, let me know.

### CODE REVIEW

**Review by chenxiaolong — CHANGES_REQUESTED** [review]
Thanks for the PR!

I'm not familiar with Romanian, but it looks like the `<plurals>` translations require one for `quantity="few"`. Right now, there are only translations for `quantity="one"` and `quantity="other"`.

```
   /home/runner/work/BCR/BCR/app/src/main/res/values-ro/strings.xml:23: Error: For locale "ro" (Romanian) the following quantity should also be defined: few (e.g. "2 zile") [MissingQuantity]
      <plurals name="pref_min_duration_desc">
```

File: app/src/main/res/values-ro/strings.xml
Comment (gilav23) [inline app/src/main/res/values-ro/strings.xml:1]: Sorry if I bother you. I have updated the strings.xml file to resolve the errors.
I wanted to translate everything, now I hope it's good. A good day.

Comment (chenxiaolong) [reply app/src/main/res/values-ro/strings.xml:1]: It looks like all of the `plurals` sections got removed. Is that what you intended?

I think the proper fix is to add the `few` translations. For example:
```xml
<plurals name="pref_min_duration_desc">
    <item quantity="one">Păstrați înregistrările de cel puțin %d secundă.</item>
    <item quantity="few"> (this is the missing translation) </item>
    <item quantity="other">Păstrați înregistrările de cel puțin %d secunde.</item>
</plurals>
```

**Comment by gilav23** [comment]
Thanks for the suggestion.
As I said, I want to translate everything, but I looked at other translations (fr and en), and I didn't find any record with the item "few".
Then I checked other translations where I found the item "few", but only in one string.
Anyway, I updated strings.xml.
A good day.

**Comment by chenxiaolong** [comment]
> As I said, I want to translate everything, but I looked at other translations (fr and en), and I didn't find any record with the item "few".

Ah, the required items are different depending on the language. Android follows the Unicode rules, which are listed in a (kind of confusing) table here: https://www.unicode.org/cldr/charts/45/supplemental/language_plural_rules.html

Romanian requires `one`, `few`, and `other`:
image

(French requires `one`, `many`, and `other`, while English only requires `one` and `other`.)

**Comment by gilav23** [comment]
I have removed the item "many", in accordance with the Unicode rules indicated by you.

**Review by chenxiaolong — APPROVED** [review]
Thanks!

**Pull request merged**